app-connector-client/server/resources/api.yaml

```diff
@@ -1,430 +1,590 @@
     delete:
+      tags:
+        - "connection"
       summary: Disconnects the mock from current kyma cluster
       responses:
         204:
           description: Disconnection was successful
+        400:
+          description: "Error"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ErrorResponse"
+
   /connection/key:
     get:
-      summary: 'Gets the private key used by the mock'
+      tags:
+        - "connection"
+      summary: "Gets the private key used by the mock"
       responses:
-        '200':
+        "200":
           description: Private key of the mock
           content:
             application/file:
-             schema:
-               type: 'string'
-               format: binary
+              schema:
+                $ref: "#/components/schemas/KeyFile"
+        "400":
+          description: "Error"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ErrorResponse"
   /connection/cert:
     get:
+      tags:
+        - "connection"
       summary: Get the client certificate for connected Kyma cluster
       responses:
-        '200':
+        "200":
           description: client certificate for connected Kyma cluster
           content:
             application/file:
-             schema:
-               type: 'string'
-               format: binary
+              schema:
+                $ref: "#/components/schemas/KeyFile"
+        "400":
+          description: "Error"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ErrorResponse"
   /events:
     post:
-      summary: 'Sends a new event to the connected Kyma cluster'
+      tags:
+        - "event"
+      summary: "Sends a new event to the connected Kyma cluster"
       requestBody:
-        description: 'Event payload'
+        description: "Event payload"
         required: true
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/EventDetails'
+              $ref: "#/components/schemas/EventDetails"
       parameters:
-      - in: 'query'
-        name: 'localKyma'
-        description: 'deployment to local kyma'
-        required: true
-        schema:
-          type: 'boolean'
+        - in: "query"
+          name: "localKyma"
+          description: "deployment to local kyma"
+          required: true
+          schema:
+            type: "boolean"
       responses:
-        '200':
-          description: 'Successful operation'
+        "200":
+          description: "Successful operation"
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/ServiceId'
-        '400':
-          description: 'Bad request'
+                $ref: "#/components/schemas/EventId"
+        "400":
+          description: "Bad request"

```

Comment:
More description please, a bad request is caused by the client and must be fixed by the client, so please point out what are typical reasons on that endpoint